email/ncmec_test.go

```diff
@@ -0,0 +1,163 @@
+package email
+
+import (
+	"strings"
+	"testing"
+	"time"
+)
+
+// TestNCMECClient contains a couple of unit tests that verify the functionality
+// of the NCMEC client.
+func TestNCMECClient(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+	t.Parallel()
+
+	// os.Setenv("NCMEC_USERNAME", "[ENTER_USERNAME]")
+	// os.Setenv("NCMEC_PASSWORD", "[ENTER_PASSWORD]")
+	// os.Setenv("NCMEC_DEBUG", "true")
```

Comment:
forgotten debug

Reply:
I kind of want to leave these in... It's like a placeholder for when you want to run the tests against the actual NCMEC (test) API

Reply:
closing this one in favour of https://github.com/SkynetLabs/abuse-scanner/pull/12#discussion_r834241200